pkgs/development/compilers/glslang/base.nix

```
@@ -0,0 +1,46 @@
+{ version, src, spirv-tools, spirv-headers }:
+{ stdenv, fetchFromGitHub, fetchpatch, cmake, bison, jq, python }:
+let
+own-src = src;
+spirv-tools-src = fetchFromGitHub ({
+  owner = "KhronosGroup";
+  repo = "SPIRV-Tools";
+} // spirv-tools);
+spirv-headers-src = fetchFromGitHub ({
+  owner = "KhronosGroup";
+  repo = "SPIRV-Headers";
+} // spirv-headers);
+in stdenv.mkDerivation rec {
+  name = "glslang-${version}";
+  inherit version;
+
+  src = fetchFromGitHub ({
+    owner = "KhronosGroup";
+    repo = "glslang";
+  } // own-src);
+
+  buildInputs = [ cmake python bison jq ];
```

All of those looks like build-time dependencies in which case:

```suggestion
  nativeBuildInputs = [ cmake python bison jq ];
```